.github/workflows/publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish
+
+on:
+  push:
+    branches: [ "main" ]
+
+jobs:
+  sbt:
+    name: Publish to apache nexus
+    runs-on: ubuntu-20.04
+    if: github.repository == 'apache/incubator-pekko-sbt-paradox'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 100
+
+      - name: Set up JDK 8
+        uses: olafurpg/setup-scala@v13
+        with:
+          java-version: adopt@1.8
+
+      - name: Publish
+        run: |-
+          sbt publish
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
```

Comment:
Is this is just copied from somewhere else or are you intended to release signed artifacts for now? Just asking because its not exactly clear we will be releasing signed artifacts via CI (this is especially the case if the plugin will do a proper release).

Reply:
It wouldn't hurt to sign with some key, but let's not for now.